### PR TITLE
Add commcarehq.org suffix

### PR DESCRIPTION
**Pull request by benrudolph**

### CODE REVIEW

File: touchforms/backend/touchcare.py [hunk 1]
```diff
@@ -53,7 +53,7 @@ def __init__(self, sessionvars, auth, restore_xml=None,
         self.uses_sqlite = uses_sqlite
         if self.uses_sqlite:
             username = sessionvars['username']
-            self.username = username + '@' + sessionvars['domain'] if '@' not in username else username
+            self.username = username + '@' + sessionvars['domain'] + '.commcarehq.org' if '@' not in username else username
```
Comment (dimagimon) [inline touchforms/backend/touchcare.py]: E501 line too long (123 > 115 characters)

**Comment by benrudolph** [comment]
@wpride

**Comment by benrudolph** [comment]
bump

**Comment by wpride** [comment]
sorry bud

**Pull request merged**